src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss

```diff
@@ -1,5 +1,7 @@
 @import "scss/constants.scss";
 
+$no-events-margin-top: 100px;
```

Comment:
Odd number to have, more so that it's combined with auto and 0

Reply:
To center an element, what usually works is to give the parent:
```css
  .parent {
    display: grid;
    place-content: center;  /* one of those place- props */
  }
```

Reply:
got rid of this margin in favor of `flex`
grid didn't really work for me here 3c4ede1